app/src/main/java/com/infomaniak/mail/ui/MainActivity.kt

```diff
@@ -443,6 +450,19 @@ class MainActivity : BaseActivity() {
         }
     }
 
+    private fun handleInAppReview() = with(localSettings) {
+        if (showAppReviewDialog && appLaunches != 0 && appLaunches % 50 == 0) {
+            titleDialog.show(
+                title = R.string.reviewAlertTitle,
+                onPositiveButtonClicked = {
+                    showAppReviewDialog = false
+                    launchInAppReview()
+                },
+                onNegativeButtonClicked = { openUrl(getString(R.string.urlUserReportAndroid)) },
```

Comment:
Do we instantly stop the users in their flow, without asking, by sending them out of the app and in their browser if they clic on 'no'? They just started the app with probably a thing in mind they had to do, maybe a good amount of them simply don't want to interact with us and just want to quickly go check what they came here for

Reply:
Seen on kChat.